lms/djangoapps/grades/migrations/0001_initial.py

```diff
@@ -2,19 +2,42 @@
 from __future__ import unicode_literals
```

Comment:
Meaning, others would also need to do the same migration rollback as required on prod - but without data loss.

But I can see if there's a relatively easy way to avoid the trouble, we should do so.  But I want to make sure we don't allow default non-empty course-ids going forward.

Sent from my iPhone

> On Sep 13, 2016, at 12:17 PM, Nimisha @ EdX <nasthagiri@edx.org> wrote:
> 
> The use of this table is behind a feature flag.  So it should not affect anyone else on devstack/sandbox.
> 
> Sent from my iPhone
> 
>> On Sep 13, 2016, at 10:46 AM, Kevin Falcone <notifications@github.com> wrote:
>> 
>> In lms/djangoapps/grades/migrations/0001_initial.py:
>> 
>> > @@ -2,19 +2,42 @@
>> >  from __future__ import unicode_literals
>> Isn't squashing going to cause anyone with a devstack/sandbox or anyone close following to have serious migration shenanigans? This isn't limited to us in production.
>> 
>> Does django let you add courseid with '' as the default and in the next migration drop it?
>> 
>> Two larger suggestions from the team (including @fredsmith who'd be doing these shenanigans) are to either drop and recreate the table, or pivot the name of the model. But I suggest looking at the extra migration first.
>> 
>> —
>> You are receiving this because you modified the open/close state.
>> Reply to this email directly, view it on GitHub, or mute the thread.
>>